src/plugins/gui/amountQuotePlugin.ts

```diff
@@ -523,8 +523,13 @@ export const amountQuoteFiatPlugin: FiatPluginFactory = async (params: FiatPlugi
               quoteFiatCurrencyCode === 'iso:USD' ? '1' : (await getHistoricalRate(`${quoteFiatCurrencyCode}_iso:USD`, new Date().toISOString())).toString()
             const quoteDollarValue = mul(bestQuote.fiatAmount, quoteFiatRate)
 
-            if (gt(quoteDollarValue, '50')) {
-              stateManager.update({ statusText: { content: lstrings.fiat_plugin_purchase_limit_error, textType: 'error' } })
+            if (gt(quoteDollarValue, DEFAULT_FIAT_AMOUNT_LIGHT_ACCOUNT)) {
```

Comment:
I see this can work since in the previous commit you floored the last decimal place instead of rounding, but this is not clean logic. This is prone to break in the future if anyone touches either code location without knowing about the other. The logic of each don't even equal each other. The other commit takes $50 converted to the fiat currency then floors it. This code takes the EXACT exchange rate conversion.

Better to create a shared function that both places can call. Something like

`getDefaultFiatValue(startingAmount, isoFiatCurrencyCode, isoNow) => fiatAmount`

Reply:
It was intentional to have the exact exchange rate shown because that's ultimately what we check against. We want users to be able to buy as much as possible, but for the initial value we make it look nice with a rounder number

Reply:
It's a big ugly to show a value that's down to the penny. Just match the actual default fiat amount which can both go up or down based on `round` instead of `floor`